src/base/name/dname.rs

```diff
@@ -181,6 +182,42 @@ impl<Octs> Dname<Octs> {
     {
         unsafe { Self::from_octets_unchecked(b"\0".as_ref().into()) }
     }
+
+    /// Creates a domain name for reverse IP address lookup.
+    ///
+    /// The returned name will use the standard suffixes of `in-addr.arpa.`
+    /// for IPv4 addresses and `ip6.arpa.` for IPv6.
+    pub fn from_addr(addr: IpAddr) -> Result<Self, PushError>
```

Comment:
I think it would be good to have "reverse" in the name somewhere as I misunderstood at first glance thinking oh is this going to resolve the address to a name for me?

Reply:
How about `reverse_from_addr`?